jekyll-site-src/images/component_icons/bottom_app_bar.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M2 2v20h20V2H2zm18 18H4v-3h5.14c.63.9 1.68 1.5 2.86 1.5s2.23-.6 2.86-1.5H20v3zm-8-3.5c-.83 0-1.5-.67-1.5-1.5s.67-1.5 1.5-1.5 1.5.67 1.5 1.5-.67 1.5-1.5 1.5zm3.5-1.5c0-1.93-1.57-3.5-3.5-3.5S8.5 13.07 8.5 15H4V4h16v11h-4.5z"/><path fill="none" d="M0 0h24v24H0z"/></svg>
```

Comment:
Oops. Posted on the deleted file.

Please remove `<path fill="none" d="M0 0h24v24H0z"/>`